### PR TITLE
Add print Class

### PR DESCRIPTION
**Pull request by mohamadhase**

### CODE REVIEW

File: modules/Print.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                    "{") and line != "" and not line.__contains__("}"):
+                line_number = self.data.lines.index(line) + 1
+                pprint(
+                    f" Unreachable Code detected in line number :  {line_number} "
+                )
+                pprint(f"    {line} ")
+                pprint("_" * 45)
+        pprint("+" * 60)
+
+    def __print_paramerters(self):
```
Comment (motaz-hashhoush) [inline modules/Print.py:48]: where comments?

**Review by yaseen-asaliya — APPROVED** [review]

File: modules/UnReachableCode.py [hunk 2]
```diff
@@ -124,44 +99,71 @@ def codeTrace(self)->None:
...
 
-                    if (line.__contains__("}")):
-                        in_if = False
+            for line in block.split("\n"):
+
+                if return_detected == True:
+                    if line not in self.result:
+                        self.result.append(line)
+
+                if line.__contains__("return ") and Number_of_blocks == 0:
```
Comment (motaz-hashhoush) [inline modules/UnReachableCode.py:162]: don't use '__contains__' function use key worde 'in'

File: modules/UnReachableCode.py [hunk 3]
```diff
@@ -124,44 +99,71 @@ def codeTrace(self)->None:
...
+                    return_detected = True
+
+                if line.__contains__("if") or line.__contains__("else"):
+                    Number_of_blocks += 1
+
+                if line.__contains__("}"):
```
Comment (motaz-hashhoush) [inline modules/UnReachableCode.py:168]: fuck u 💖

**Review by motaz-hashhoush — CHANGES_REQUESTED** [review]

File: modules/Print.py (hunk 1 above)
Comment (mohamadhase) [reply modules/Print.py:48]: in your ass

File: modules/UnReachableCode.py (hunk 3 above)
Comment (mohamadhase) [reply modules/UnReachableCode.py:168]: الله يسعدك حبيبي

File: modules/UnReachableCode.py (hunk 2 above)
Comment (mohamadhase) [reply modules/UnReachableCode.py:162]: done

**Review by motaz-hashhoush — APPROVED** [review]

**Pull request merged**